app/services/attachment_renderer.rb

```diff
@@ -19,7 +19,7 @@ def render_one
     return unless record.persisted?
 
     attachment = record.send(attachment_name)
-    return unless !attachment.respond_to?(:attached?) || attachment.attached?
+    return unless !attachment.respond_to?(:attached?) || (attachment.attached? && attachment.persisted?)
```

Comment:
what's this about, I didn't see anything related to this in the [upgrade guide](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-7-0-to-rails-7-1)